### PR TITLE
Use radio button for renderer

### PR DESCRIPTION
**Pull request by RisingGeek**
Changed `<select>` tags with `<input>` with type of radio button.

### CODE REVIEW

File: src/components/sidebar/renderer.tsx [hunk 1]
```diff
@@ -84,22 +84,29 @@ class Sidebar extends Component<any, any> {
...
-              isSearchable={false}
-              onChange={e => this.props.setRenderer(e.label)}
+            <input
+              type="radio"
+              name="renderer"
+              value="canvas"
+              defaultChecked
+              onClick={e => this.props.setRenderer(e.currentTarget.value)}
+            />
+            <label>canvas</label>
```
Comment (domoritz) [inline src/components/sidebar/renderer.tsx]: Please move the input into the label and capitalize the renderer name.

File: src/components/sidebar/renderer.tsx [hunk 2]
```diff
@@ -84,22 +84,29 @@ class Sidebar extends Component<any, any> {
...
+          </div>
+          <div>
+            <input
+              type="radio"
+              name="renderer"
+              value="svg"
+              onClick={e => this.props.setRenderer(e.currentTarget.value)}
             />
+            <label>svg</label>
```
Comment (domoritz) [inline src/components/sidebar/renderer.tsx]: Same as above

**Comment by RisingGeek** [comment]
> When I change the renderer and reload the page, the renderer is reset. The radio button should be set based on the renderer in the redux store.

Working on this

**Comment by RisingGeek** [comment]
@domoritz I have fixed the issue but the travis ci build failed. I am new to travis ci. In the logs, there's a module not found error in spec file. What can I do for that?

**Comment by siddhant1** [comment]
Build is failing even on master , maybe some other commit broke it

**Comment by RisingGeek** [comment]
> Build is failing even on master , maybe some other commit broke it

Oh, I thought my changes broke the build.

**Comment by domoritz** [comment]
I just fixed the build.

**Comment by domoritz** [comment]
Awesome. Thank you. I merged it manually.

**Comment by RisingGeek** [comment]
> Awesome. Thank you. I merged it manually.

Welcome!

**Pull request closed without merging**